modules/cardinal_service_profile_helper/templates/components/cardinal-quarter-grid/cardinal_quarter_grid.html.twig

```diff
@@ -58,7 +60,16 @@
     {% endif %}
 
     <div class="su-opportunity-grid-location">
-      <h4>{{ 'Location'|t }}:</h4> <div>{{ location }}</div>
+      <h4>{{ 'Location'|t }}:</h4>
+      {% set locations = location['#markup'] | split('\n') %}
```

Comment:
can't you do this with field formatter instead?
![image](https://github.com/user-attachments/assets/b33c6d26-b732-41fb-96f1-a094ec35e34a)

Reply:
@pookmish Okay. We'll do that.